tests/suites/test_suite_ecp.function

```diff
@@ -122,12 +122,14 @@ void ecp_test_vect_restart( int id,
     mbedtls_mpi dA, xA, yA, dB, xZ, yZ;
     int cnt_restarts;
     int ret;
+    mbedtls_test_rnd_pseudo_info rnd_info;
```

Comment:
Why do some tests use `mbedtls_test_rnd_pseudo_rand` and others use `mbedtls_test_rnd_std_rand`?

Reply:
That's purely a historical accident. When editing this file, I looked around and used the same thing as other tests in this file. When editing another file, I did the same and noticed `std_rand` was used. Then in later commits I just used `std_rand` all the time as that meant I didn't need to declare and initialize a `rnd_pseudo_info` context.

(I tend to forget what test RNGs we have, and I think we probably have more than we need, but that's for another day.)